examples/webpack/package.json

```diff
@@ -5,8 +5,8 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "webpack": "^4.10.2",
-    "webpack-cli": "^2.1.4",
+    "webpack": "^4.20.2",
+    "webpack-cli": "^3.1.1",
```

Comment:
According to https://www.npmjs.com/package/webpack-cli, `3.1.2` is the latest stable version, so this could just read `^3.1.2` instead (assuming obviously that version 3 works in general).